frontend/src/pages/_recruitment/admin/GrantAccessDialog.js

```diff
@@ -76,8 +89,9 @@ export default ({ userId, onClose }) => {
                     onChange={setSelectedEvents}
                     isMulti={true}
                     options={events.map(event => ({
-                        value: event._id,
+                        value: event,
```

Comment:
- [ ] Optional: Here the value property is receiving the entire object, and then on line 61 is deconstructed from the event object captured on this Select component. It would be better to only pass the event._id without having to run the .map method on the array returned by the Select component

Reply:
Both event._id and event.slug are needed for different dispatch operations. Also because it is admin side (should not be in major use in the future), I am not concerned about performance